cmd/api.go

```diff
@@ -0,0 +1,29 @@
+package cmd
+
+import (
+	"net/http"
+
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/thirdweb-dev/indexer/internal/handlers"
+)
+
+var (
+	apiCmd = &cobra.Command{
+		Use:   "api",
+		Short: "TBD",
+		Long:  "TBD",
```

Comment:
```suggestion
		Short: "a",
		Long:  "api",
```

Reply:
These are descriptions for the command, not shorthand syntax